Software/app/src/main/java/org/foi/emp/collegecoursemanager/DodavanjeKolegija.java

```diff
@@ -0,0 +1,101 @@
+package org.foi.emp.collegecoursemanager;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.view.Display;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import org.foi.emp.core.Database.Database;
+import org.foi.emp.core.Entities.ElementModelaPracenja;
+import org.foi.emp.core.Entities.ElementiNaModeluPracenja;
+import org.foi.emp.core.Entities.Kolegij;
+import org.foi.emp.core.Entities.ModelPracenja;
+
+public class DodavanjeKolegija extends AppCompatActivity {
+    Button spremi;
+    EditText nazivKolegija;
+    EditText modelPracenja;
+    EditText kolokvij1, kolokvij2, aktivnost, labosi;
```

Comment:
This are what? package-private, public, private? Annotate variables private if they don't need to be seen outside of the class. If they do, annotate public.